web/src/beta/lib/reearth-ui/components/PopupMenu/index.tsx

```diff
@@ -0,0 +1,122 @@
+import { FC, useCallback, useState } from "react";
+import { Link } from "react-router-dom";
+
+import { Popup, Icon, Typography, IconName } from "@reearth/beta/lib/reearth-ui";
+import { styled, useTheme } from "@reearth/services/theme";
+
+const MULTLEVEL_OFFSET = 6;
+const DEFAULT_OFFSET = 4;
+const DEFAULT_MENU_WIDTH = 180;
+
+export type Items = {
+  title?: string;
```

Comment:
```suggestion
 id: string;
  title?: string;
```
We need a unique id for each item, so that when emit onClick we can carry this id and user can know which one is been clicked.